### ISSUE TITLE
Use of CLIQ with proxy (process goes into loop)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [LeoLiXX/dataloadercliq#11]
```
What steps will reproduce the problem?
1. Started the process clicking bat file 
2. Selected the Operation and provided the process name
3. Clicked to Next button

What is the expected output? What do you see instead?

Process takes time to redirect user to the Login screen


What version of the product are you using? On what operating system?
CLIq- Latest on Window


Please provide any additional information below.




```

Original issue reported on code.google.com by `rohan62...@gmail.com` on 13 Jan 2010 at 1:21

**Comment by GoogleCodeExporter** [comment]
```
Hi Rohan,
About how long did it take to redirect?
Were you able to use it successfully after the delay?

Vijay
```

Original comment by `vijay.swamidass@gmail.com` on 13 Jan 2010 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Please try the 2.0 version of CLIq - it should be much more reliable.
```

Original comment by `vijay.swamidass@gmail.com` on 19 Oct 2011 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in CLIQ 2.0.  Please let me know if not.
```

Original comment by `vijay.swamidass@gmail.com` on 13 Jan 2012 at 7:13
- Changed state: **Fixed**
